DarkTube.css

```diff
@@ -1188,6 +1188,32 @@ rect[fill="#cccccc"] {
 #exp-cont {
     background-color: #131313;
     }
+.yt-pl-thumb .popoutmode,
+.thumb-wrapper .popoutmode,
+.yt-lockup-thumbnail .popoutmode,
+.yt-pl-thumb .blacklist,
+.thumb-wrapper .blacklist,
+.yt-lockup-thumbnail .blacklist {
+    background-color: #2A2A2A;
+    border: 1px solid #1C1C1C;
+    border-radius: 2px;
+    color: #C3C3C3;
+    cursor: pointer;
+    line-height: 1;
+    position: absolute;
+    z-index: 99;
+    width: 0;

```

Comment:
width, height, padding, margin, etc. are not needed at all ;)
but thanks for your first pull request! :D